docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.hero.scss

```diff
@@ -40,33 +38,47 @@
     margin: calc(-1 * #{hb-spacing-width()}) 0 hb-spacing-width();
   }
 
-  @include hb-colorful {
-    position: relative;
-    min-height: 60vh;
+  position: relative;
```

Comment:
can you move these styles up by the classname (hb-hero-overlay) - just seems a bit odd sitting in the middle here